### ISSUE TITLE
Column role "weight"

### ISSUE THREAD

**Issue by sebffischer** [mlr-org/mlr3#1016]
Currently it only says "Observation weights. Not more than one numeric column may have this role." in the Task documentation. 
Also, I got the reasonable question whether these are also applied during resampling.

**Comment by sebffischer** [comment]
There is a very old and open PR regarding weights for resampling: https://github.com/mlr-org/mlr3/pull/239

**Comment by sebffischer** [comment]
Also it was asked whether the weights are applied when scoring predictions via measures.

**Comment by berndbischl** [comment]
> Currently it only says "Observation weights. Not more than one numeric column may have this role." in the Task documentation.

is there a question related to this?

**Comment by berndbischl** [comment]
* we will improve docs to make clear what currently happens. what currently happens is that these weights are handed down to each learner, in training, when the learner supports a weighted fit. if the learners doesn't support weights, they are ignored

**Comment by berndbischl** [comment]
* we can also have the weights being used in measures

```
task$set_col_role("my_weights", "weights")
my_learner = lrn("regr.rpart", use_weights = TRUE) # ???
my_resa = rsmp("cv")
rr = resample(task, my_learner, my_resa)
mm = msr("regr.mse", task_weights = TRUE)
rr$score(mm)
```
this control where weights are used

**Comment by berndbischl** [comment]
in task: col_roles: weights_train, weights_measures
maybe: weights_resampling

**Comment by berndbischl** [comment]
we also want flags in learner and measure, to control wether the weights are used in computation. also we need to ensure that the docs are clear. maybe the book

**Comment by berndbischl** [comment]
closed by PR #1065
